### ISSUE TITLE
Site not available?

### ISSUE THREAD

**Issue by pmackay** [Code-for-All/civicpatterns.org#36]
@pudo do you know who maintains the domain?

**Comment by pudo** [comment]
I used to have it, but it's expired. Can I suggest we just re-mount it to civicpatterns.github.io?

**Comment by almereyda** [comment]
Yes, please do it. Civic patterns are a treasure trove and I would hate to see them disappear from the webs.

To gain that URL, there is a need to have https://github.com/civicpatterns/civicpatterns.github.io existing, if I am not mistaken?

Also the other site is detoriating, where the style doesn't hold up anymore.

- http://codeforamerica.github.io/civic-tech-patterns/

Are these two projects related? If even a little, it becomes imaginable that they could share that new @civicpatterns organisation, and both publish from there, growing a wider community.

Until then I'll go with the last working snapshot http://web.archive.org/web/20210413012243/https://civicpatterns.org/.

—

It also appears the domain had been/is squatted in the meantime. http://web.archive.org/web/20211127034027/https://www.civicpatterns.org/patterns/ask-forgiveness-not-permission

And one can use the current site at http://civicpatterns.org/ to get one's own public IP address:

```
Parse error: syntax error, unexpected '<' in wp-config.php : eval()'d code on line 51 - 88.134.22.229 fake
```

**Comment by futuresoup** [comment]
😢  still down. Was hoping to use it today and perhaps contribute.
